projects/addon-table/components/table/th/th.style.less

```diff
@@ -124,6 +124,10 @@
     &:hover {
         color: var(--tui-text-01);
     }
+
+    &-icon {
```

Comment:
Let's just rename it `.t-icon`, no need for it to be nested. Strange that we had that class in HTML with no styles applied to it before.

Reply:
renamed